src/main/java/tr/edu/ogu/ceng/controller/SettingController.java

```diff
@@ -0,0 +1,37 @@
+package tr.edu.ogu.ceng.controller;
+
+import javax.persistence.EntityNotFoundException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import tr.edu.ogu.ceng.model.Setting;
+import tr.edu.ogu.ceng.service.SettingService;
+
+@RestController
+@RequestMapping("/api/setting")
+public class SettingController {
+	@Autowired
+	private SettingService settingService;
+
+	@PutMapping
+	public ResponseEntity<Setting> updateSetting(@RequestBody Setting setting) {
+	    try {
+	        Setting updatedSetting = settingService.updateSetting(setting);
+	        return ResponseEntity.ok(updatedSetting);
+	    } 
+	    catch (EntityNotFoundException ex) {
```

Comment:
Exception yakalamıyoruz hiç bir zaman

Reply:
Hocam bu pr'i merglesek olur mu? Exception yakalama için Issue oluşturalım. Şuan çalışamıyoruz.

Reply:
Bir hatayı başka bir hata ile kapatmak çok doğru değil.

Reply:
https://github.com/esogu-ceng/internship-management-system/pull/96 merge olunca problemi çözmese de derleme problemi düzelecek.